### PR TITLE
[Term Entry] Java Calendar internalGet() (#3102)

### LINKED ISSUE #3102: [Term Entry] Java Calendar internalGet()
### Reason/inspiration (optional)

Please add the `internalGet()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar)�class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/internalGet/internalGet.md`

The entry should have the following:

�  An introductory paragraph about what the method is and does.
�  A syntax section with a pseudo code block with an explanation of it.
�  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by LeoMiguelB**
### Description

This PR includes an entry to the Calendar class for the internalGet method. It includes an example and a description. The path for this new entry is `content/java/concepts/calendar/terms/internalGet/internalGet.md.`

Closes #3102 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [✔️] All writings are my own.
- [ ✔️] My entry follows the Codecademy Docs style guide.
- [✔️ ] My changes generate no new warnings.
- [✔️ ] I have performed a self-review of my own writing and code.
- [✔️ ] I have checked my entry and corrected any misspellings.
- [ ✔️] I have made corresponding changes to the documentation if needed.
- [✔️ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✔️ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✔️ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/internalGet/internalGet.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+Title: '.internalGet()'
+Description: 'Returns the value of a given field'
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/internalGet/internalGet.md:3]:
```suggestion
Description: 'Returns the value of a given field.'
```

File: content/java/concepts/calendar/terms/internalGet/internalGet.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.internalGet()`** method is a protected method used to retrieve the value of a field from a Calendar instance. In other words it takes a field as input and returns its corresponding value. This method is generally used within the Calendar class or its sub classes.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/internalGet/internalGet.md:14]:
```suggestion
The **`.internalGet()`** method is a protected method used to retrieve the value of a field from a `Calendar` instance. In other words, it takes a field as input and returns its corresponding value. This method is generally used within the `Calendar` class or its sub classes.
```

File: content/java/concepts/calendar/terms/internalGet/internalGet.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Syntax
+
+```pseudo
+protected final int internalGet(int field)
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/internalGet/internalGet.md:19]:
```suggestion
myCalendar.internalGet(int field)
```

File: content/java/concepts/calendar/terms/internalGet/internalGet.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+```
+
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/internalGet/internalGet.md:21]: There should be a note here after the syntax regarding the parameters taken, return value, or any other valuable information

File: content/java/concepts/calendar/terms/internalGet/internalGet.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+## Example 1
+
+In this example, the `InternalGetExample` class inherits the `GregorianCalendar` class which is a concrete subclass of the Calendar class. The code first instantiates a new object of type `InternalGetExample` with the current date which uses the Gregorian calendar system.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/internalGet/internalGet.md:24]:
```suggestion
In this example, the `InternalGetExample` class inherits the `GregorianCalendar` class. The code first instantiates a new object of type `InternalGetExample` with the current date which uses the Gregorian calendar system.
```

File: content/java/concepts/calendar/terms/internalGet/internalGet.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+The **`.internalGet()`** method is a protected method used to retrieve the value of a field from a Calendar instance. In other words it takes a field as input and returns its corresponding value. This method is generally used within the Calendar class or its sub classes.
+
+## Syntax
+
+```pseudo
+protected final int internalGet(int field)
+```
+
+## Example 1
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/internalGet/internalGet.md:22]:
```suggestion
## Example 
```

File: content/java/concepts/calendar/terms/internalGet/internalGet.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+In this example, the `InternalGetExample` class inherits the `GregorianCalendar` class which is a concrete subclass of the Calendar class. The code first instantiates a new object of type `InternalGetExample` with the current date which uses the Gregorian calendar system.
+
+> **Note:** There are multiple calendar systems that `Calendar.getInstance()` provides depending on the locale. These include `GregorianCalendar`, `BuddhistCalendar`, and `JapaneseImperialCalendar`.
+
+```java
+import java.util.GregorianCalendar;
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/internalGet/internalGet.md:29]: update the code to use 2 spaces for indentation instead of tabs

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@LeoMiguelB I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by LeoMiguelB** [comment]
@caupolicandiaz, I followed your suggestions and corrected my entry.

**Review by caupolicandiaz — APPROVED** [review]
@LeoMiguelB I made a couple of additional edits directly. This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@LeoMiguelB Looks good.

**Pull request merged**